ovs/extensions/packages/packagefactory.py

```diff
@@ -17,14 +17,18 @@
 """
 Package Factory module
 """
+from ovs_extensions.db.arakoon.pyrakoon.pyrakoon.compat import ArakoonException
 from ovs.extensions.generic.configuration import Configuration
+from ovs.extensions.generic.logger import Logger
 from ovs_extensions.packages.packagefactory import PackageFactory as _PackageFactory
 
 
 class PackageFactory(_PackageFactory):
     """
     Factory class returning specialized classes
     """
+    _logger = Logger('package_factory')
```

Comment:
We tend to use the 'extensions' logger in the extensions
You could use 'extensions-packages' to use the extensions logger with a specification to the packages